app-backup/snapper/snapper-0.5.4.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils systemd
+
+DESCRIPTION="Command-line program for btrfs and ext4 snapshot management"
+HOMEPAGE="http://snapper.io/"
+SRC_URI="ftp://ftp.suse.com/pub/projects/snapper/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+btrfs ext4 lvm pam xattr"
+
+RDEPEND="dev-libs/boost:=[threads]
+	dev-libs/libxml2
+	dev-libs/icu:=
+	sys-apps/acl
+	sys-apps/dbus
+	sys-apps/util-linux
+	sys-libs/zlib
+	virtual/libintl
+	btrfs? ( >=sys-fs/btrfs-progs-3.17.1 )
+	ext4? ( sys-fs/e2fsprogs )
+	lvm? ( sys-fs/lvm2 )
+	pam? ( sys-libs/pam )
+	xattr? ( sys-apps/attr )"
+
+DEPEND="${RDEPEND}
+	sys-devel/gettext
+	virtual/pkgconfig"
+
+REQUIRED_USE="|| ( btrfs ext4 lvm )"
+
+DOCS=( AUTHORS )
+
+PATCHES=(
+	"${FILESDIR}"/cron-confd.patch
+)
+
+src_prepare() {
+	default
+
+	# Fix systemd services and timers installation path
+	sed -i "s,/usr/lib/systemd/system,$(systemd_get_systemunitdir),g" data/Makefile.*
```

Comment:
missing `|| die`, but I can fix this while merge.

Reply:
thanks to push it